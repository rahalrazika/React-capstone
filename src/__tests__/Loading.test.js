import React from 'react';
import renderer from 'react-test-renderer';
import Loading from '../components/loading';

it('expect to render loading components', () => {
  const result = renderer.create(<Loading />).toJSON;
  expect(result).toMatchSnapshot();
});
