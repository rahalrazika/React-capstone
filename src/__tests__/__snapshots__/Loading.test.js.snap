// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render loading components 1`] = `[Function]`;
